utils/update_checkout/update-checkout-config.json

```diff
@@ -367,7 +367,7 @@
                 "sourcekit-lsp": "swift-DEVELOPMENT-SNAPSHOT-2020-08-04-a",
                 "PythonKit": "master",
                 "swift-format": "master",
-                "tensorflow": "v2.2.0-rc0",
+                "tensorflow": "v2.3",
```

Comment:
oh actually I'm not sure if there's a "v2.3" tag on https://github.com/tensorflow/tensorflow/tags. It might have to be "v2.3.0"